test/integration/preflight/preflight_test.go

```diff
@@ -70,8 +104,8 @@ func (p PlaybookTest) checkExitCode(t *testing.T, got, want int, cmd *exec.Cmd,
 	if got == want {
 		return
 	}
-	t.Fatalf("got exit code %v, want %v", got, want)
 	p.logCmdAndOutput(t, cmd, output)
+	t.Fatalf("got exit code %v, wanted %v", got, want)
```

Comment:
that may be, but is it too much to ask for the two verbs to be in the same tense? :)